media/server/gstplayer/source/tasks/AttachSource.cpp

```diff
@@ -80,16 +80,16 @@ std::string createCapsStr(std::shared_ptr<firebolt::rialto::server::IGstWrapper>
         case firebolt::rialto::SegmentAlignment::NAL:
             return "video/x-h264, stream-format=byte-stream, alignment=nal";
         default:
-            return "video/x-h265";
+            return "video/x-h264";
         }
     }
     else if (mimeType.compare("video/mpeg2") == 0)
     {
         return "video/mpeg, mpegversion=(int) 2";
     }
-    else if (mimeType.compare("ideo/x-theora") == 0)
+    else if (mimeType.compare("video/x-theora") == 0)
```

Comment:
Can you profile out any MIME types not listed here please https://www.stb.bskyb.com/confluence/pages/viewpage.action?pageId=167854321

We'll enable them later if/when required